frontend/packages/process-editor/src/components/Canvas/Canvas.tsx

```diff
@@ -40,11 +42,29 @@ export const Canvas = ({ onSave }: CanvasProps): JSX.Element => {
   );
 };
 
-
 // Below is helper components for Canvas.tsx
-const Viewer = (): JSX.Element => {
-  const { canvasRef } = useBpmnViewer();
-  return <div ref={canvasRef}></div>;
+export const Viewer = (): JSX.Element => {
+  const { t } = useTranslation();
+  const { canvasRef, renderNoDiagramError, renderNoProcessError } = useBpmnViewer();
+  return (
+  <>
+  {(renderNoDiagramError || renderNoProcessError) && ( 
+  <div className={classes.alert} >
+   <Alert severity="warning">
+    <Heading size='small' data-testid={"no_diagram"}>
+      {renderNoDiagramError && (t("process_editor.not_found_diagram_heading"))}  
+      {renderNoProcessError && (t("process_editor.not_found_process_heading"))}
+    </Heading>
+    <Paragraph className={classes.paragraph} data-testid={"no_process"}> 
+      {renderNoDiagramError && (<Trans i18nKey={'process_editor.not_found_diagram_error_message'} components={{ a: <Link>her</Link> }}/>)}  
+      {renderNoProcessError && (<Trans i18nKey={"process_editor.not_found_process_error_message"} components={{ a: <Link>her</Link> }}/>)}
+    </Paragraph>
+   </Alert>
+  </div>
+  )}
```

Comment:
We can simplify the JSX/markup by sperate the logic to build error message away from the markup.😊 We can also remove some custom CSS by using the built-in spacing prop on the Heading component. 😎 

```suggestion
  {errorToDisplay && (
  <div className={classes.alertContainer}>
     <Alert severity='warning'>
        <Heading size='small' spacing>
          {errorToDisplay.heading}
        </Heading>
        <Paragraph>{errorToDisplay.body}</Paragraph>
     </Alert>
  </div>
```

The errorToBeDisplay can be like this:

```typescript
const getErrorMessage = (): { heading: string; body: string } | null => {
    if (renderNoDiagramError) {
      return {
        heading: t('process_editor.not_found_diagram_heading'),
        body: t('process_editor.not_found_diagram_error_message'),
      };
    }
    if (renderNoProcessError) {
      return {
        heading: t('process_editor.not_found_process_heading'),
        body: t('process_editor.not_found_process_error_message'),
      };
    }
    return null;
  };

  const errorToDisplay = getErrorMessage();
```